test/e2e/wizard.test.js

```diff
@@ -98,84 +98,104 @@ describe('vl-wizard', async () => {
     });
 
     it('als gebruiker kan ik via de actie knoppen alleen door de panelen van de wizard navigeren wanneer ik de checkbox aanzet', async () => {
```

Comment:
dacht da k da al eens ergens opgemerkt had, maar "als ik de checkbox aanzet", zegt niet over wat ge functioneel kunt met de wizard en is iets specifieks van de demo-pagina, dus lijkt me niet echt een correcte beschrijving van de functionaliteit

Reply:
Die opmerking is in PR #32 opgelost, dus kan daar eventueel nog nagekeken worden.